pico/api/src/main/java/io/helidon/pico/Services.java

```diff
@@ -58,7 +58,7 @@ public interface Services {
      * @throws io.helidon.pico.PicoException if resolution fails to resolve a match
      */
     default <T> ServiceProvider<T> lookup(Class<T> type) {
-        return lookupFirst(type, true).get();
+        return lookupFirst(type, true).orElseThrow();
```

Comment:
`orElseThrow` or `get` throw a very useless exception. In these cases, we may want to use a custom exception describing what you tried to do and why it failed.
This can be a follow up issue.

Reply:
it was changed to address an IntelliJ warning.  Since lookupFirst(type, true) is being used here, returning an optional will/should never occur - like assert this is just a sanity check on an insane condition.  And it removes the intellij warnings in this style.

Reply:
I think you are still missing my point about it being an insane condition to check for, but this is minor and will just do what you suggest in the next commit.

Reply:
pushed